python_modules/libraries/dagster-dg/dagster_dg/cli/dev.py

```diff
@@ -76,6 +75,12 @@
     default=False,
     help="Show verbose stack traces, including system frames in stack traces.",
 )
+@click.option(
+    "--no-check-yaml",
```

Comment:
For bool flags, rather than having only a negative variant, nice to do:

`"--check-yaml/--no-check-yaml"` (that's special syntax click understands) and default to True. This is better for if we put this in a config file in a future.